sw/device/lib/base/status.h

```diff
@@ -104,6 +104,44 @@ typedef struct status {
     status_.value;                         \
   })
 
+/**
+ * Evaluates a status_t for Ok or Error status, returning the Ok value.
+ *
+ * This macro is like the `try!` macro (or now `?` operator) in Rust:
```

Comment:
`try!` has been deprecated for a while and most people will be familiar with `?` now. How do you feel about removing the reference to `try!`? I know it came from the other doc comment

Reply:
Done for the new macro, mas I won't change the existing comment in this PR.